src/components/mongo.ts

```diff
@@ -0,0 +1,409 @@
+import * as pulumi from '@pulumi/pulumi';
```

Comment:
I think it might be a good idea to extract mongo/webserver common logic to EcsService and reuse that service for mongo and webserver services in the scope of this PR because there is a lot of repetitive logic.